doc/src/build/move/sui-move-library.md

```diff
@@ -70,16 +70,14 @@ transfer::freeze_object(obj);
 ```
 After this call, `obj` becomes immutable which means it can never be mutated or deleted. This process is also irreversible: once an object is frozen, it will stay frozen forever. An immutable object can be used as reference by anyone in their Move call.
 
-### Share an object (experimental)
-This feature is still in development. It only works in Move for demo purpose, and doesn't yet work in Sui.
-
+### Share an object
 To make an object `obj` shared and mutable, one can call:
 ```
 transfer::share_object(obj);
 ```
-After this call, `obj` stays mutable, but becomes shared by everyone, i.e. anyone can send a transaction to mutate this object. However, such an object cannot be deleted, transferred or embedded in another object as a field.
+After this call, `obj` stays mutable, but becomes shared by everyone, i.e. anyone can send a transaction to mutate this object. However, such an object cannot be deleted, transferred or embedded in another object as a field. For an example, see [Shared Object](https://examples.sui.io/basics/shared-object.html#shared-object) on https://examples.sui.io/.
```

Comment:
```suggestion
After this call, `obj` stays mutable, but becomes shared by everyone, i.e. anyone can send a transaction to mutate this object. However, such an object cannot be transferred or embedded in another object as a field. For more detail, see the [shared objects](../../learn/build/objects.md#shared) documentation.
```

- I moved the more thorough description of shared objects to the "objects" section and simply linked to that description here
- We will soon (but do not yet) support shared object deletion, so I think we can avoid talking about deletion for now + explicitly mention that is supported once it lands.

Reply:
Thanks, Sam!